### PR TITLE
mahage-2022.01.31-eDiscovery-additionalData-additionalDataOptions

### PR DESCRIPTION
**Pull request by mahage-msft**

### CODE REVIEW

**Review by kapilratra02 — APPROVED** [review]

File: api-reference/beta/api/ediscovery-reviewset-addtoreviewset.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph.ediscovery
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Start the process of adding a collection from Microsoft 365 services to a review set. After the operation is created, you can get the status of the operation by retrieving the `Location` parameter from the response headers. The location provides a URL that will return a [caseExportOperation](../resources/ediscovery-caseexportoperation.md).
+Start the process of adding a collection from Microsoft 365 services to a review set. After the operation is created, you can get the status of the operation by retrieving the `Location` parameter from the response headers. The location provides a URL that will return a [addToReviewSetOperation](../resources/ediscovery-addToReviewSetOperation.md).
```
Comment (Lauragra) [inline api-reference/beta/api/ediscovery-reviewset-addtoreviewset.md]: Why doesn't the example show response headers with a URL?

File: api-reference/beta/resources/ediscovery-addtoreviewsetoperation.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ None.
 |resultInfo|[resultInfo](../resources/resultinfo.md)|Contains success and failure-specific result information. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md).|
 |status|[microsoft.graph.ediscovery.caseOperationStatus](../resources/ediscovery-caseoperation.md#caseoperationstatus-values)|The status of the case operation. Inherited from [caseOperation](../resources/ediscovery-caseoperation.md). Possible values are: `notStarted`, `submissionFailed`, `running`, `succeeded`, `partiallySucceeded`, `failed`.|
 
-### dataCollectionScope values
+### additionalDataOptions values
```
Comment (Lauragra) [inline api-reference/beta/resources/ediscovery-addtoreviewsetoperation.md:37]: Which property has this enum as its type?

File: changelog/Microsoft.Compliance.Ediscovery.json [hunk 3]
```diff
@@ -362,6 +362,32 @@
 			"WorkloadArea": "Compliance",
 			"SubArea": "eDiscovery"
 		},
+		{
+			"ChangeList": [
```
Comment (Lauragra) [inline changelog/Microsoft.Compliance.Ediscovery.json:366]: This ChangeList item doesn't make any sense; please update it to reflect the changes in this PR.

**Comment by Lauragra** [comment]
@mahage-msft , please take a look at this PR and incorporate feedback. The schema associated with this PR has been released so we should get the docs ready to publish. Thanks.

**Pull request closed without merging**